tests/python/gaia-ui-tests/gaiatest/gaia_test.py

```diff
@@ -117,8 +117,13 @@ def displayed_app(self):
                        origin=result.get('origin'))
 
     def switch_to_displayed_app(self):
+        from gaiatest.apps.lockscreen.app import LockScreen
+
         self.marionette.switch_to_default_content()
```

Comment:
Apparently, this is the same as self.marionette.switch_to_frame(). Perhaps you could change it to use that?